src/moonstone-samples/lib/TimePickerSample.js

```diff
@@ -83,7 +84,7 @@ module.exports = kind({
 	},
 	timeChanged: function (sender, event) {
 		if (this.$.result && event.value){
-			var timeArray = event.value.toTimeString().split(': ');
+			var timeArray = event.value.toTimeString().split(':');
```

Comment:
Yes you are right. I just keep consistency with 2.5-upkeep.
Actually we don't need split it anymore